### DISCUSSION TITLE
RGB LEDs on Arduino Mega

### DISCUSSION THREAD

**Discussion by ghost** [shanteacontrols/OpenDeck#61] [General]
Hi there,
I'm having some troubles getting RGB LEDs to work with Arduino mega2560 (latest firmware flashed) 
The RGB led works fine when the board is set to individual LEDs so all the connections are good and correct, but when switched to RGB the test LED function no longer works. Is this function fully supported with these boards or did i miss something else?

I also have an official Opendeck V1.2.2 board here, which seems to work fine with the same configuration. 

Thanks again for your hard work with this, and any advice would be much appreciated.

**Comment by paradajz** [comment]
Can you tell me which pins exactly do you have the RGB LED connected to, and also, which RGB LED did you enable in the UI?

**Reply by ghost** [reply]
Pins 17,16 and 15 Enabled on LED 0. 
I've gone through each of the connections on this table and enabling the corresponding RGB but each give no output when RGB is enabled.
Set to single LED

 rather than RGB the same pins work. 
Thanks.
Robin.
![rgb](https://user-images.githubusercontent.com/7473427/115558229-32cfdf00-a2bb-11eb-9085-85f05f682e83.PNG)
![configurator](https://user-images.githubusercontent.com/7473427/115559426-60695800-a2bc-11eb-985d-ad671dca7618.PNG)
![duino](https://user-images.githubusercontent.com/7473427/115560252-1339b600-a2bd-11eb-9dcd-9c830d4d8dcb.jpg)

**Comment by ghost** [comment]
Here's the messages sent when testing the LED's
![system](https://user-images.githubusercontent.com/7473427/115560752-8fcc9480-a2bd-11eb-8112-80d3c1a8449c.PNG)

**Reply by paradajz** [reply]
This is a bug. I've fixed it and will release new version later today.

**Reply by paradajz** [reply]
v5.4.5 is out: https://github.com/paradajz/OpenDeck/releases/tag/v5.4.5

**Comment by ghost** [comment]
Thanks! Legend.Sent from my Galaxy
-------- Original message --------From: paradajz ***@***.***> Date: 27/04/2021  18:17  (GMT+02:00) To: paradajz/OpenDeck ***@***.***> Cc: robinsutherland ***@***.***>, Author ***@***.***> Subject: Re: [paradajz/OpenDeck] RGB LEDs on Arduino Mega (#61) 
v5.4.5 is out: https://github.com/paradajz/OpenDeck/releases/tag/v5.4.5
